### PR TITLE
feat: add navigation links to header and footer

### PR DESCRIPTION
**Pull request by simboonlong**
### User story

https://www.pivotaltracker.com/story/show/172105807
https://www.pivotaltracker.com/story/show/172119033

---

### Design

https://www.figma.com/file/NrlglAZw3v0kZcgpAkbBi8/TradeTrust---Website?node-id=886%3A0

---

### PR updates

- added relevant links to header.
- show header links in mobile.
- remove drawer component from header.
- implement "sticky" footer, with copyright text.
- updated faq content.
---

# Before
### Header (no hamburger for header links in mobile)
header

### Drawer (This is removed, as it will block the header links hamburger)
drawer

### CODE REVIEW

File: src/components/Layout/Footer.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+              <a href={item.path} target={item.target} rel="noopener">
+                {item.label}
+              </a>
+            </li>
+          );
+        })}
+      </ul>
+      <div className="row justify-content-center">
+        <div className="col-auto">
+          <p>Copyright &copy; 2019 All rights reserved</p>
```
Comment (sumitnitsurat) [inline src/components/Layout/Footer.js]: 2020 it is ?

**Comment by sumitnitsurat** [comment]
I dont think your design for github link matches with the story . please check story #172118915

File: src/components/Layout/Footer.js (hunk 1 above)
Comment (simboonlong) [reply src/components/Layout/Footer.js]: think is 2019 still according to design and pivotal, if need change then we change it again later on?

**Comment by sumitnitsurat** [comment]
one more thing...the FAQ content is not correct.  it uses opencerts at few of the places. Better to get it from Serena or confirm it from her.

**Comment by simboonlong** [comment]
> one more thing...the FAQ content is not correct. it uses opencerts at few of the places. Better to get it from Serena or confirm it from her.

yup okay, gotten the latest content from her and updated

**Review by sumitnitsurat — APPROVED** [review]
Lgtm. Just the navigation at the header is bit weird. some link goes outside website and links randomly move to different pieces/sections of the website.

**Pull request merged**